File: ACT1.1.java
import java.util.Scanner;

class Student{
    private String name;
    private int age;
    private String program;
    private int yearLevel;
    private String course;
    
    public Student(String name, int age, String program, int yearLevel, String course){
        this.name = name;
        this.age = age;
        this.program = program;
        this.yearLevel = yearLevel;
        this.course = course;
    }
    
    public String getName(){
        return name;
    }
    
    public int getAge(){
        return age;
    }
    
    public String getProgram(){
        return program;
    }
    
    public int getYearLevel(){
        return yearLevel;
    }
    
    public String getCourse(){
        return course;
    }
    
    public void displayStudInfo(){
        System.out.println("Name: " + name);
        System.out.println("Age: " + age);
        System.out.println("Program: " + program);
        System.out.println("Year Level: " + yearLevel);
        System.out.println("Course: " + course);
    }
}

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter your name: ");
        String name = scanner.nextLine();
        System.out.print("Enter your age: ");
        int age = scanner.nextInt();
        scanner.nextLine();
        System.out.print("Enter your program: ");
        String program = scanner.nextLine();
        System.out.print("Enter your Year Level: ");
        int yearLevel = scanner.nextInt();
        scanner.nextLine();
        System.out.print("Enter your course: ");
        String course = scanner.nextLine();
        
        Student stud = new Student(name, age, program, yearLevel, course);
        
        stud.displayStudInfo();
    }
}

File: ACT1.1.py
# Create a program that defines a Student class using full object-oriented programming principles. The class should encapsulate relevant student details and include functionality to display them. Instantiate and utilize the class in a separate section of the program to simulate object usage.
# Name: Stanley
# Age: 20
# Program: BSCS
# Year Level: 1
# Course: OOP

class Student:
    def __init__(self, name, age, program, year_level, course):
        self.__name = name
        self.__age = int(age)
        self.__program = program
        self.__year_level =     year_level
        self.__course = course
    
    def get_name(self):
        return self.__name
    
    def get_age(self):
        return self.__age
        
    def get_program(self):
        return self.__program
        
    def get_year_level(self):
        return self.__year_level
    
    def get_course(self):
        return self.__course
        
    def display_stud_info(self):
        print(f"Name: {self.__name}")
        print(f"Age: {self.__age}")
        print(f"Program: {self.__program}")
        print(f"Year Level: {self.__year_level}")
        print(f"Course: {self.__course}")
        
if __name__ == "__main__":
    name = input("Enter your name: ")
    age = int(input("Enter your age: "))
    program = input("Enter your program: ")
    year_level = input("Enter your year level: ")
    course = input("Enter your course: ")
    
    stud = Student(name, age, program, year_level, course)
    
    stud.display_stud_info()
    
